src/core/server/core_usage_data/types.ts

```diff
@@ -27,21 +27,91 @@ import { ISavedObjectTypeRegistry, SavedObjectTypeRegistry } from '..';
  * includes point-in-time configuration information.
  * */
 export interface CoreUsageStats {
+  // Saved Objects Client APIs
+  'apiCalls.savedObjectsBulkCreate.total'?: number;
+  'apiCalls.savedObjectsBulkCreate.namespace.default.total'?: number;
+  'apiCalls.savedObjectsBulkCreate.namespace.default.kibanaRequest.yes'?: number;
+  'apiCalls.savedObjectsBulkCreate.namespace.default.kibanaRequest.no'?: number;
+  'apiCalls.savedObjectsBulkCreate.namespace.custom.total'?: number;
+  'apiCalls.savedObjectsBulkCreate.namespace.custom.kibanaRequest.yes'?: number;
```

Comment:
I guess this reflects the ES schema? Don't we want nested properties instead of flattening everything?

Reply:
Yes this reflects the schema. In the original PR I had nested properties, but I flattened the fields so we could use the `incrementCounter` API to increment the counter fields instead of `get`+`update`. At any rate, the fields are flattened inside of ES so it doesn't make a difference for how we'll consume the usage data 🙂